### PR TITLE
Save immutable config parameters into configmap

### PR DESCRIPTION
**Pull request by legionus**

### CODE REVIEW

File: Dockerfile-dev [hunk 1]
```diff
@@ -4,6 +4,6 @@ RUN useradd cluster-image-registry-operator
 USER cluster-image-registry-operator
 
 COPY cluster-image-registry-operator /usr/bin
-COPY deploy/image-references deploy/crd.yaml deploy/namespace.yaml deploy/operator.yaml deploy/rbac.yaml /manifests/
+COPY deploy/image-references deploy/*.yaml /manifests/
```
Comment (bparees) [inline Dockerfile-dev]: does the dev image need this at all?  fundamentally if you're not going to use the image w/ the installer, the /manifests dir doesn't matter.

**Comment by bparees** [comment]
/lgtm

you can remove the COPY command in a follow up if you want, or leave it there for consistency.

File: Dockerfile-dev (hunk 1 above)
Comment (legionus) [reply Dockerfile-dev]: @bparees I added this just in case. Suddenly someday in the future it will be needed. This is not difficult to copy.

Comment (bparees) [reply Dockerfile-dev]: ok, that's fine

**Pull request merged**